include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -978,6 +1003,10 @@ class CommonOperatorLoader : public ProtobufLoader {
       RETURN_IF_ERR(loadRelu(op, dict));
       return true;
     }
+    if (typeName == "PRelu") {
```

Comment:
Since this is in CommonOperatorLoader it will be run on c2 nets as well, have you tested it with any c2 nets?

Reply:
No. It should though (How come so many operators are not shared between onnx and c2?). I can add models and tests for c2.